tests/test-sources/plugins/languages/orgmode.nix

```
@@ -0,0 +1,16 @@
+{
+  empty = {
+    plugins.orgmode.enable = true;
+  };
+
+  example = {
```

```suggestion
  defaults = {
```

Correct me if I'm wrong, but this appears to be the "plugin defaults" test case?

You are of course welcome to _also_ have an "example" test-case alongside "defaults". :grin:

---

See https://github.com/nix-community/nixvim/pull/2115#discussion_r1739916923